content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -76,16 +79,17 @@ $ curl http://localhost:8080/api/
 
 ### Without kubectl proxy
 
-Use `kubectl describe secret...` to get the token for the default service account:
-
-Use `kubectl describe secret` with grep/cut:
+Use `kubectl describe secret...` to get the token for the default service account with grep/cut:
 
 ```shell
-$ APISERVER=$(kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")
-$ SECRET_NAME=$(kubectl get secrets | grep ^default | cut -f1 -d ' ')
-$ TOKEN=$(kubectl describe secret $SECRET_NAME | grep -E '^token' | cut -f2 -d':' | tr -d " ")
+APISERVER=$(kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")
+SECRET_NAME=$(kubectl get secrets | grep ^default | cut -f1 -d ' ')
+TOKEN=$(kubectl describe secret $SECRET_NAME | grep -E '^token' | cut -f2 -d':' | tr -d " ")
 
-$ curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+```
+
```

Comment:
please add The output is similar to this: as per the guidelines before any output